src/analytics/index.ts

```diff
@@ -1,10 +1,31 @@
 import { createClient } from '@segment/analytics-react-native';
 // @ts-ignore
 import { REACT_APP_SEGMENT_API_WRITE_KEY } from 'react-native-dotenv';
+import { MMKV } from 'react-native-mmkv';
+import { STORAGE_IDS } from '@rainbow-me/model/mmkv';
 
-export const analytics = createClient({
+const mmkv = new MMKV();
+
+const analyticsClient = createClient({
   debug: false,
   trackAppLifecycleEvents: true,
   trackDeepLinks: true,
   writeKey: REACT_APP_SEGMENT_API_WRITE_KEY,
 });
+
+const identify = (userId: string | null, traits: Record<string, any>) => {
```

Comment:
set up a basic wrapper, this will ensure all of our current calls handle current uid properly and leaves room for sanitizing etc.